### PR TITLE
Feature/examples

### PR DESCRIPTION
**Pull request by ColdForeign**
## Checklist

- [ ] Documentation updated
- [ ] Tests cover new or modified code
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] New dependencies added
- [ ] Includes breaking changes
- [ ] Version bumped

### CODE REVIEW

File: src/Cropper.Blazor/Client/Services/MenuService.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
+﻿using Cropper.Blazor.Client.Models;
+
+namespace Cropper.Blazor.Client.Services
+{
+    public interface IMenuService
```
Comment (MaxymGorn) [inline src/Cropper.Blazor/Client/Services/MenuService.cs:5]: move IMenuService to anothrt file

File: src/Cropper.Blazor/Client/Styles/layout/_markdown.scss [hunk 2]
```diff
@@ -121,8 +120,7 @@
 
     & .csharp {
         & .atSign {
-            color: var(--mud-palette-text-primary);
```
Comment (MaxymGorn) [inline src/Cropper.Blazor/Client/Styles/layout/_markdown.scss:124]: Why we make changes for this file?

Comment (ColdForeign) [reply src/Cropper.Blazor/Client/Styles/layout/_markdown.scss:124]: active buttons styles fix

File: src/Cropper.Blazor/Cropper.Blazor.Client.Compiler/Cropper.Blazor.Client.Compiler.csproj [hunk 3]
```diff
@@ -1,13 +1,16 @@
...
+		<TargetFramework>net7.0</TargetFramework>
+	</PropertyGroup>
 
   <ItemGroup>
     <ProjectReference Include="..\Cropper.Blazor.Shared\Cropper.Blazor.Shared.csproj" />
     <ProjectReference Include="..\Cropper.Blazor\Cropper.Blazor.csproj" />
   </ItemGroup>
 
+	<ItemGroup>
+		<PackageReference Include="ColorCode.HTML" Version="2.0.14" />
```
Comment (MaxymGorn) [inline src/Cropper.Blazor/Cropper.Blazor.Client.Compiler/Cropper.Blazor.Client.Compiler.csproj:14]: remove this package

Comment (ColdForeign) [reply src/Cropper.Blazor/Cropper.Blazor.Client.Compiler/Cropper.Blazor.Client.Compiler.csproj:14]: It is used in the ExmaplesMarkup.cs file

**Review by MaxymGorn — DISMISSED** [review]

**Review by MaxymGorn — APPROVED** [review]

**Pull request merged**